css-positioning/absolute.css

```diff
@@ -0,0 +1,39 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+  bottom: 20px;
+  left: 5px;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: relative;
+  bottom: 100px;
+  left: 220px;
```

Comment:
These are **_big_** movements. Instead of pushing this all the way from the `left` and `bottom`, you should use `top` and `right`